### PR TITLE
enh(chore): backend code owners

### PR DESCRIPTION
**Pull request by sc979**
## Description

Keep only backend owners

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1,9 @@
-.github/**/*.yml    @centreon/owners-pipelines
-
-*.po                @centreon/owners-doc
-*.md                @centreon/owners-doc
-*.mdx               @centreon/owners-doc
+.github/**          @centreon/owners-pipelines
```
Comment (cg-tw) [inline .github/CODEOWNERS:2]:
```suggestion

*.po @centreon/owners-doc

```

Comment (cg-tw) [reply .github/CODEOWNERS:2]: we still need to be automatically added to PRs that contain .po files

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (sc979) [reply .github/CODEOWNERS:2]: there's no po files in this repository !!!

Comment (cg-tw) [reply .github/CODEOWNERS:2]: indeed. my bad.

**Pull request merged**